cassovary-core/src/main/scala/com/twitter/cassovary/algorithms/PageRank.scala

```diff
@@ -42,7 +42,7 @@ object PageRank {
    * @param params PageRankParams from above
    * @return An array of doubles, with indices corresponding to node ids
    */
-  def apply(graph: DirectedGraph, params: PageRankParams): Array[Double] = {
+  def apply[V <: Node](graph: DirectedGraph[V], params: PageRankParams): Array[Double] = {
```

Comment:
No need to parametrize in this file. It uses only node.id that you can get by just Directedgraph[Node]